Source/IntegrationTests/TestFiles/LitTests/LitTest/c++/tuple.dfy.expect

```diff
@@ -1,3 +1,3 @@
 
-Dafny program verifier finished with 6 verified, 0 errors
+Dafny program verifier finished with 7 verified, 0 errors
```

Comment:
Do you know why this changed? I don't think it's likely a problem, but it's odd that just changing to the new CLI would do this.

Reply:
Oh, it's due to the change in default for definite assignment, isn't it?

Reply:
Could be, but also the previous number was amount of implementations (a concept almost invisible to the user), and the new number is amount of assertion batches (slightly visible).